### DISCUSSION TITLE
jxls-reader latest version

### DISCUSSION THREAD

**Discussion by vishy-wisy** [jxlsteam/jxls#211] [Q&A] [answered]
Hi Team
As per current maven repo latest jxls-reader version is  2.0.6 which is not compatible with apache poi 5.x . 

Do you have any plan to upgrade jxls-reader with latest apache poi version . 

I am using 2.0.5 in my project and after upgrading apache poi OffsetRowCheckImpl.java is failing because method getCellTypeEnum is changed to getCellType . 

Kindly let me know if you have plan to upgrade or is there any way to handle this scenario . 

Thanks in advance .

**Comment by SoltauFintel** [comment]
see https://github.com/jxlsteam/jxls-reader/issues/3

@leonate would have to answer the rest.

**Reply by vishy-wisy** [reply]
@SoltauFintel  Thanks for your reply . 
Does it mean , the upgrade is planned and already fixed but not released it and leonate will share the release plan . 

@leonate would you please help with the date . 
I completely understand this is hobby project and supported on best effort basis . Kindly let me know is there any way I can contribute .

**Reply by SoltauFintel** [reply]
It would be easiest if you built the master version yourself. I can't help further here.

**Reply by leonate** [reply]
@vishy-wisy I have not looked into this myself yet. I can try to look into it in the next few weeks and try to bump the compatibility.
Contributions are always welcome, feel free to create PR if you want to speed it up.

**Reply by vishy-wisy** [reply]
@leonate  , Thanks for your reply . I can see PR already raised https://github.com/jxlsteam/jxls-reader/issues/3 .  I am happy to contribute in any form , kindly let me know how I can assist .

**Reply by leonate** [reply]
Oh, indeed, great! So I guess it should work on `master` branch. Can you check if the jxls-reader `master` work for you?
If so I can just make a release to make it official.

**Reply by vishy-wisy** [reply]
@leonate : yes it working for me . But do we have any kind of regression testing suite which we can run before making it official .

**Reply by leonate** [reply]
Sure, we have a set of unit tests and a few integration tests that are executed for new commits. I do not remember what is the coverage level though.

**Comment by leonate** [comment, marked as answer]
@vishy-wisy I released `jxls-reader-2.1.0` so it should become avalaible in the Maven central soon.

**Reply by vishy-wisy** [reply]
@leonate thanks a lot
